pkgs/development/python-modules/vosk/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, srt
+, requests
+, tqdm
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "vosk";
+  version = "0.3.45";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    dist = "py3";
+    platform = "manylinux_2_12_x86_64.manylinux2010_x86_64";
+
+    hash = "sha256-JeAlCTxDmdcnj1Q1aO2MxUYKw6S/SMI2c6zh4l0mYZ8=";
+  };
+
+  propagatedBuildInputs = [
+    srt
+    requests
+    tqdm
+    websockets
+  ];
+
+  meta = with lib; {
+    description = "Offline open source speech recognition toolkit";
+    mainProgram = "vosk-transcriber";
```

Comment:
Does it provide executables, or is it just a library, or both?

Reply:
it does provide an executable

Reply:
Ok, I tried running `vosk-transcriber` executable, and `ffmpeg`is missing, some other non-python runtime dependencies might also be missing.

Running tests from GitHub repo might reveal some dependencies.